Tests/appstoreconnect-cliTests/Operations/GetUserInfoOperationTests.swift

```diff
@@ -0,0 +1,71 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+@testable import AppStoreConnectCLI
+import AppStoreConnect_Swift_SDK
+import Combine
+import Foundation
+import XCTest
+
+typealias Dependencies = GetUserInfoOperation.Dependencies
+typealias Options = GetUserInfoOperation.Options
+typealias OperationError = GetUserInfoOperation.GetUserInfoError
+
+final class GetUserInfoOperationTests: XCTestCase {
+    func testCouldNotFindUserError() {
+        let dependencies: Dependencies = .noUsers
+        let options = Options(email: "bob@bob.com")
+        let operation = GetUserInfoOperation(options: options)
+
+        let expectation = XCTestExpectation(description: "Publisher will complete")
+        var operationError: Error?
+
+        _ = operation
+            .execute(with: dependencies)
+            .sink(
+                receiveCompletion: { completion in
+                    if case .failure(let error) = completion {
+                        operationError = error
+                    }
+
+                    expectation.fulfill()
+                },
+                receiveValue: { _ in
+                    XCTFail("Expected no values")
+                }
+            )
+
+        XCTAssertEqual(
+            operationError?.localizedDescription,
+            OperationError.couldNotFindUser(email: "bob@bob.com").localizedDescription
+        )
+    }
+}
+
+private extension Dependencies {
```

Comment:
```suggestion
private extension GetUserInfoOperationTests.Dependencies {
```

Reply:
The type alias is actually at file scope, did you want me to move the alias into the test class and extend that?

Reply:
Just saw the comment above 👍